jaeger-core/src/main/java/com/uber/jaeger/Tracer.java

```diff
@@ -262,12 +304,15 @@ private SpanContext createChildContext(SpanContext preferredParent) {
           return preferredParent;
         }
       }
+
       return new SpanContext(
           preferredParent.getTraceId(),
           Utils.uniqueId(),
+          // for zipkin compatibility
```

Comment:
Comment is not accurate. We capture parent ID not because of zipkin, but as storage optimization for 99% use case of a plain parent-child relationship. Similar to my comment above, I think the explanation is best placed in the SpanContext class comment.